.github/workflows/nightly-packages.yml

```diff
@@ -0,0 +1,80 @@
+name: Nightly packages
+
+on:
+  workflow_dispatch:
+  schedule:
```

Comment:
We have a `cron` trigger in `packages` workflow as well.

How about adding an additonal step in `packages` about the package upload instead of copying the entire packages workflow?

We could protect the package upload by checking if the event source is `cron` and the branch is master.